concordium-consensus/src/Concordium/External.hs

```diff
@@ -517,6 +513,18 @@ stopBaker cptr = mask_ $
 +-------+------------------------------------+----------------------------------------------------------------------------------------+----------+
 |    13 | ResultConsensusShutDown            | Consensus has been shut down and the message was ignored                               | No       |
 +-------+------------------------------------+----------------------------------------------------------------------------------------+----------+
+|    14 | ResultExpiryTooLate                | The transaction expiry time is too far in the future                                   | No       |
++-------+------------------------------------+----------------------------------------------------------------------------------------+----------+
+|    15 | ResultVerificationFailed           | The transaction signature verification failed                                          | No       |
++-------+------------------------------------+----------------------------------------------------------------------------------------+----------+
+|    16 | ResultNonexistingSenderAccount     | The transaction's sender account does not exist according to the focus block           | No       |
```

Comment:
Could it be the case that our focus block is wrong, and we couldn't look up the account by mistake? Should we still retransmit transactions in that case?